src/ruby/ext/grpc/rb_channel_credentials.c

```diff
@@ -270,7 +270,13 @@ void Init_grpc_channel_credentials() {
 /* Gets the wrapped grpc_channel_credentials from the ruby wrapper */
 grpc_channel_credentials* grpc_rb_get_wrapped_channel_credentials(VALUE v) {
   grpc_rb_channel_credentials* wrapper = NULL;
+  Check_TypedStruct(v, &grpc_rb_channel_credentials_data_type);
```

Comment:
question: why do we need this call to `Check_TypedStruct`? Does this add any safety checks not performed by `TypedData_Get_Struct`?

If this is necessary, should it be added to `grpc_rb_get_wrapped_server_credentials` too?

Reply:
There is no safety check in `TypedData_Get_Struct`.
Yeah, will add to `grpc_rb_get_wrapped_server_credentials` as well